src/get-project.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = function(input) {
+  if (input.project) {
+    return input.project;
+  } else if (input.root && typeof input.isEmberCLIProject === 'function') {
```

Comment:
FWIW, I decided to use the presence of this function as a way to confirm we were given the project directly because I can easily imagine a future refactor in ember-cli that would only setup `bowerDirectory` _if_ a `bower.json` was detected...